src/pylorax/api/v0.py

```diff
@@ -1103,8 +1111,12 @@ def v0_blueprints_info(blueprint_names):
         blueprints = sorted(blueprints, key=lambda r: r["name"].lower())
 
         if out_fmt == "toml":
-            # With TOML output we just want to dump the raw blueprint, skipping the rest.
-            return "\n\n".join([r.toml() for r in blueprints])
+            if errors:
+                # If there are errors they need to be reported, use JSON and 400 for this
+                return jsonify(status=False, errors=errors), 400
```

Comment:
Wouldn't it be better to move the `if errors` block above the `if out_fmt == "toml"` block ? This will make the code a bit easier to read and also avoids these nested if conditions.

Reply:
That would break the JSON response which can return both errors and blueprints. TOML can't do that so we have to return the TOML recipes OR the JSON error response.